### PR TITLE
trim feature for vktrace

### PR DESCRIPTION
**Pull request by ardaLunarG**

### CODE REVIEW

**Comment by ardaLunarG** [comment]
Compile fails due to missing package required for hotkey:

libxcb-keysyms1 - utility libraries for X C Binding -- keysyms
libxcb-keysyms1-dev - utility libraries for X C Binding -- keysyms, development files

**Comment by ardaLunarG** [comment]
After travis update Linux start to compile w/o hassle. Android fails due to header includes. 
Peter and Ming please review patches and after your approval I will merge to master.

File: vktrace/vktrace_layer/vktrace_lib_pageguard.cpp [hunk 1]
```diff
@@ -182,8 +183,8 @@ void* pageguardAllocateMemory(size_t size)
     if (size != 0)
     {
 #if defined(WIN32)
-        pMemory = (PBYTE)VirtualAlloc(nullptr, pageguardGetAdjustedSize(size), MEM_WRITE_WATCH | MEM_RESERVE | MEM_COMMIT,
```
Comment (plohrmann) [inline vktrace/vktrace_layer/vktrace_lib_pageguard.cpp]: Was this an intended change? I know the MEM_WRITE_WATCH came in recently, so maybe it's just that the branch needs to be rebased again?

Comment (ardaLunarG) [inline vktrace/vktrace_layer/vktrace_lib_pageguard.cpp]: Not intended, it is wrong change. Thanks for catching this!

Comment (ardaLunarG) [inline vktrace/vktrace_layer/vktrace_lib_pageguard.cpp]: Done

Comment (plohrmann) [inline vktrace/vktrace_layer/vktrace_lib_pageguard.cpp]: Confirmed.

**Comment by ardaLunarG** [comment]
Merged

**Pull request closed without merging**